app/src/main/java/com/example/cinemapp/ui/main/HomeFragment.kt

```diff
@@ -61,6 +69,23 @@ class HomeFragment : Fragment() {
         }
     }
 
+    private fun onChipChanged(checked: Int) {
+        with(binding){
+            when(checked) {
+                cUpcoming.id -> viewModel.getUpcomingNextPage()
```

Comment:
Since Chip is not made up of 2 words (like TextView), we have a strange naming here 😅 maybe consider renaming this chip (and others) to have prefix `chip`